### PR TITLE
Add changelogs for April 2023 patch releases (v2.22.2, v2.21.8, v2.20.15)

### PR DESCRIPTION
**Pull request by embik**
**What this PR does / why we need it**:
This PR adds the changelogs for v2.22.2, v2.21.8 and v2.20.15, the upcoming April 2023 patch releases.

**Which issue(s) this PR fixes**:

Fixes #

**What type of PR is this?**
/kind documentation

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change? Then add your Release Note here**:

```release-note
NONE
```

**Documentation**:

```documentation
NONE
```

### CODE REVIEW

**Comment by embik** [comment]
/hold

to be merged later.

File: docs/changelogs/CHANGELOG-2.20.md [hunk 1]
```diff
@@ -15,6 +15,21 @@
...
+
+## [v2.20.15](https://github.com/kubermatic/kubermatic/releases/tag/v2.20.15)
+
+### Bugfixes
+
+- Fix calculation of node CPU utilisation in Grafana dashboards for multi-core nodes ([#12081](https://github.com/kubermatic/kubermatic/pull/12081))
+
+### Updates
+
+- [User Cluster MLA 0.2.6](https://github.com/kubermatic/mla/releases/tag/v0.2.6) has been released and requires manual updating by re-running [User Cluster MLA Installation](https://docs.kubermatic.com/kubermatic/v2.20/tutorials-howtos/monitoring-logging-alerting/user-cluster/admin-guide/#installation) ([#12106](https://github.com/kubermatic/kubermatic/pull/12106))
```
Comment (pkprzekwas) [inline docs/changelogs/CHANGELOG-2.20.md]: Shouldn't that be in `Action required` section?

Comment (embik) [reply docs/changelogs/CHANGELOG-2.20.md]: Fair point, I've updated accordingly.

**Comment by embik** [comment]
/hold cancel 

forgot to unhold this PR, releases are out.

**Pull request merged**